lopper/lops/lop-load-cfg-obj-a53.dts

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
Delete all these lops files as we can invoke them via command line these lops files are not needed.

Reply:
Resolved in latest commit